charts/cryostat/values.yaml

```diff
@@ -71,9 +71,9 @@ db:
     repository: "quay.io/cryostat/cryostat-db"
     ## @param db.image.pullPolicy Image pull policy for the database container image
     pullPolicy: Always
-    ## @param db.image.tag Tag for thedatabasestorage container image
+    ## @param db.image.tag Tag for the database container image
```

Comment:
`db.image.tag` and `db.resources` descriptions use "thedatabasestorage container" in `README.md`, `charts/cryostat/README.md`, and `charts/cryostat/values.schema.json`